src/cljs/harja/ui/liitteet.cljs

```diff
@@ -298,45 +308,57 @@
            [:progress {:value edistyminen :max 100}]
            ;; Näytetään uuden liitteen lisäyspainike
            [:span.liitekomponentti
-            [:label {:class (str "file-upload nappi-toissijainen "
+            [:button {:tabIndex "0"
+                      :id "tiedoston-lataus-label"
+                      :for "tiedoston-lataus-input"
```

Comment:
Buttonilla ei kai ole for-attribuuttia ollenkaan? Eli tuo ei tee mitään?

Reply:
Joo. Se oli jossain vaiheessa label, kun testailin sitä ja sitten kaikki toimikin ihan oikein buttonina :) 
Hyvä bongaus. Otin pois.